PowerEditor/src/WinControls/AnsiCharPanel/ansiCharPanel.cpp

```diff
@@ -38,12 +38,14 @@ intptr_t CALLBACK AnsiCharPanel::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 			generic_string charStr = pNativeSpeaker->getAttrNameStr(TEXT("Character"), "AsciiInsertion", "ColumnChar");
 			generic_string htmlNumberStr = pNativeSpeaker->getAttrNameStr(TEXT("HTML Number"), "AsciiInsertion", "ColumnHtmlNumber");
 			generic_string htmlNameStr = pNativeSpeaker->getAttrNameStr(TEXT("HTML Name"), "AsciiInsertion", "ColumnHtmlName");
+			generic_string htmlHexNbStr = pNativeSpeaker->getAttrNameStr(L"HTML Hex Number", "AsciiInsertion", "ColumnHtmlHexNb");

```

Comment:
Please swap line 40 with Line 41 to make column order as `HTML Name` `HTML Decimal` `HTML Hexadecimal`